tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 # TODO: re-enable py33,py34 when bravado-core is py3x compatible - https://github.com/Yelp/bravado-core/issues/10
-envlist = py26,py27,py33,py34,pep8
+envlist = py26,py27,py33,py34,py35,{py27,py35}-pyramid15,pep8
```

Comment:
Wow very cool. So if I'm reading right, these are the factors referred to in http://testrun.org/tox/latest/config.html#factors-and-factor-conditional-settings and basically say "make a special pair of environments with py27 and py35 where we install an old version of pyramid"?

And I guess we rely on tox installing these deps before it installs the package itself to use this version over the setup.py one?